frontend/js/i18n/de-lang.json

```diff
@@ -179,15 +179,14 @@
 		"user": "Benutzer:in"
 	},
 	"settings": {
-		"default-site": "Standard-Seite",
-		"default-site-404": "Dead Seite",
+		"default-site-404": "Seite nicht gefunden",
```

Comment:
it is correct that it sends a 404 error, but it does not a 404 error in the html file, but dead host, so there should be a better translation for "dead host